### PR TITLE
Cleanup: Correct annotation array initializer indents from checkstyle #8083

### PR DESCRIPTION
**Pull request by Abhishek-kumar09**
## Describe the PR

Its a checkstyle indentation changes for annotation array initialization that is coming with merging PR https://github.com/checkstyle/checkstyle/pull/8083

## Related issues

No related issue in your repo. Checkstyle's ci follows your project and its config for testing the latest changes, they were having violations due to the annotation arrays being not indented according to config.

### CODE REVIEW

File: pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/cpd/testdata/discardedElements.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
     @AnnotationWithParams({@Nested(1) ,
                            @Nested(2) ,
                            @Nested
-             })
+        })
```
Comment (Abhishek-kumar09) [inline pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/cpd/testdata/discardedElements.java:32]: They could be 4 or 8 indents.

File: pmd-core/src/test/java/net/sourceforge/pmd/renderers/RenderersTests.java [hunk 2]
```diff
@@ -15,18 +15,18 @@
...
-        EmacsRendererTest.class,
-        HTMLRendererTest.class,
-        IDEAJRendererTest.class,
-        PapariTextRendererTest.class,
-        SummaryHTMLRendererTest.class,
-        TextPadRendererTest.class,
-        VBHTMLRendererTest.class,
-        XMLRendererTest.class,
-        XSLTRendererTest.class,
-        YAHTMLRendererTest.class
```
Comment (Abhishek-kumar09) [inline pmd-core/src/test/java/net/sourceforge/pmd/renderers/RenderersTests.java:29]: https://github.com/pmd/build-tools/blob/a847dc52ccdaafff9b8324da4390bc2043f6e926/src/main/resources/net/sourceforge/pmd/pmd-checkstyle-config.xml#L230-L233
In your pmd config the arrayInitIndent, basicOffset, lineWrapping is 4 and thus these lines should be indented at level 8.

**Comment by pmd-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="true">No java rules are changed!</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by oowekyala — APPROVED** [review]
Thanks, this looks fine

**Pull request merged**